build

```
@@ -1,3 +1,18 @@
 #!/usr/bin/env sh
 
+set -e
+set -x
+
 clang -Ofast -Wall -Wextra -Werror -pedantic -std=c99 term-size.c -o term-size
+
+# $SIGN should be the signing identity whose key is available in the keychain,
+# e.g. "Node.js Foundation"
+if [ "X$SIGN" == "X" ]; then
+	echo "No SIGN environment variable, skipping codesigning" >&2
+else
+  codesign \
+	--sign "$SIGN" \
```

SIGN seems a bit too generic. I would go with what Xcode uses.
```suggestion
	--sign "$CODE_SIGN_IDENTITY" \
```